src/datanode/src/config.rs

```diff
@@ -218,13 +216,8 @@ pub struct DatanodeOptions {
     pub node_id: Option<u64>,
     pub require_lease_before_startup: bool,
     pub init_regions_in_background: bool,
-    pub rpc_addr: String,
     pub rpc_hostname: Option<String>,
```

Comment:
The rpc_hostname is not a part of RPC options. Should we move it to the `GrpcOptions`?

Reply:
I think so, but only the datanode use it. I don't know if I should refactor it.

Reply:
It should be fine. We can document this in the code.

Reply:
It looks like `rpc_hostname` is only used to report to metasrv through heartbeat. What's the usage of this field, can we remove it? @fengjiachun @sunng87

Reply:
We cannot. It's a fatal feature because metasrv can only get the IP address from heartbeat. Also datanode has no idea of the hostname that it will use for intra-cluster communication without specified.

Reply:
We can't remove `rpc_home`, but I think we can move it to grpcOptions.